graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/DXGraphQLProvider.java

```diff
@@ -215,6 +225,21 @@ public void activate() {
 
         extensionsProviders.add(this);
 
+        registerSchema(extensionsHandler);
```

Comment:
I don't understand the need of this specific signature, the behaviour seems to be exactly the same as calling `registerSchema()` ?